src/module.c

```diff
@@ -970,6 +970,21 @@ void RM_ChannelAtPosWithFlags(RedisModuleCtx *ctx, int pos, int flags) {
     res->numkeys++;
 }
 
+/* Return the wall-clock Unix time, in microseconds */
+long long RM_Time() {
```

Comment:
Given that we already have `RM_Milliseconds()` and `RM_MonotonicMicroseconds()`, isn't it more natural to name this `RM_Microseconds()`? and `RM_CachedMicroseconds()`?

Reply:
i agree